spec/rubocop/cop/style/hash_except_spec.rb

```diff
@@ -225,6 +261,18 @@
       RUBY
     end
 
+    it 'does not register an offense when using more than two block arguments' do
+      expect_no_offenses(<<~RUBY)
+        {foo: 1, bar: 2, baz: 3}.reject { |k, v, o| k == :bar }
+      RUBY
+    end
+
+    it 'does not register an offense when calling `include?` method without a param' do
+      expect_no_offenses(<<~RUBY)
+        {foo: 1, bar: 2, baz: 3}.reject { |k, v| %i[foo bar].include? }
+      RUBY
+    end
```

Comment:
This kind of code will break at runtime with ArgumentError, but I've added this test because the [current code](https://github.com/rubocop/rubocop/blob/5b8a474ec19b6bd81c1991d9dbb43dca5706dd74/lib/rubocop/cop/style/hash_except.rb#L108-L118) uses safe nav for the receiver and the first argument.

However, I'm not sure if safe nav is required because of the way node pattern is written (i.e., there's always a receiver and an argument) and because when I remove safe nav no tests break, so this test might be useless if save nav provides no value in current code.